pkgs/top-level/kodi-packages.nix

```diff
@@ -26,7 +26,7 @@ let
   self = {
     addonDir = "/share/kodi/addons";
 
-    rel = "Nexus";
+    rel = "Omega";
```

Comment:
Do you think it would be advantageous to rename `rel` to `release`? It took me quite a while to figure that out (when looking at the packages).

Reply:
i believe that's what it was called when i got here
i don't particularly mind one way or the other

Reply:
yes, both still needed: